wolfcrypt/src/asn.c

```diff
@@ -33071,13 +33103,25 @@ static int ParseCRL_CertList(DecodedCRL* dcrl, const byte* buf,
 
     checkIdx = idx;
     if (GetASNTag(buf, &checkIdx, &tag, sz) == 0 && tag == ASN_INTEGER) {
-        if (GetMyVersion(buf, &idx, &version, sz) < 0)
+        if (GetMyVersion(buf, &idx, &dcrl->version, sz) < 0)
             return ASN_PARSE_E;
+        dcrl->version++;
     }
 
     if (GetAlgoId(buf, &idx, &oid, oidIgnoreType, sz) < 0)
         return ASN_PARSE_E;
 
+    checkIdx = idx;
+    if (GetSequence(buf, &checkIdx, &length, sz) < 0) {
+        return ASN_PARSE_E;
+    }
+#ifdef OPENSSL_EXTRA
+    else {
+        dcrl->issuerSz = length + 3;
```

Comment:
How did you come up with `+ 3`? Is that including the sequence? Why not `checkIdx - idx`? Why does this have an `else {` here? I don't think the else is needed.

Reply:
Yes it was including the sequence, fixed. Yeah the else is completely useless, removed.